rust/src/lib.rs

```diff
@@ -0,0 +1,61 @@
+#[macro_use(lazy_static)]
+extern crate lazy_static;
+
+pub mod do_username {
+	use rand::Rng;
+	fn to_title_case(s: &str) -> String {
+		let mut c = s.chars();
+		match c.next() {
+			None => String::new(),
+			Some(f) => f.to_uppercase().chain(c).collect(),
+		}
+	}
+	
+	fn random(arr: Vec<&str>) -> &str {
+		let mut rng = rand::thread_rng();
+		let r = rng.gen_range(0, arr.len());
+		arr[r]
+	}
+	fn up_format(arr: Vec<&str>) -> String {
```

Comment:
```suggestion

	fn up_format(arr: Vec<&str>) -> String {
```
Consistent whitespace